### PR TITLE
Update to hyper 0.14 and hyper-tls 0.5

### PR DESCRIPTION
**Pull request by Fedott**
This change is [Reviewable](https://reviewable.io/reviews/panicbit/fcm-rust/19)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -19,13 +19,13 @@ serde_json = "1"
 erased-serde = "0.3"
 serde_derive = "1"
 futures = "0.3"
-hyper = "0.13"
-hyper-tls = "0.4"
+hyper = { version = "0.14", features = ["http1", "http2", "stream"] }
```
Comment (pimeys) [inline Cargo.toml:22]: Do we really need both: `http1` and `http2` support?

**Review by pimeys — COMMENTED** [review]
Just a small nitpick. Otherwise looks good and we could cut a new release. Thank you.

File: Cargo.toml (hunk 1 above)
Comment (Fedott) [reply Cargo.toml]: You are right. http2 is not needed.

File: src/client/mod.rs [hunk 2]
```diff
@@ -22,7 +22,7 @@ impl Client {
     /// Get a new instance of Client.
     pub fn new() -> Client {
         let mut http_client = HttpClient::builder();
-        http_client.keep_alive(true);
+        http_client.pool_max_idle_per_host(std::usize::MAX);
```
Comment (extrawurst) [inline src/client/mod.rs:25]: I wonder why we need this? `usize::MAX` is the default anyway

**Pull request merged**